tests/src/test/scala/whisk/core/controller/test/PackagesApiTests.scala

```diff
@@ -750,7 +750,6 @@ class PackagesApiTests extends ControllerTestCommon with WhiskPackagesApi {
       status should be(Conflict)
       val response = responseAs[ErrorResponse]
       response.error should include("Package not empty (contains 1 entity)")
-      response.code.id should be >= 1L
```

Comment:
Can you change this check to assert the id is not empty? This weakens the test to where a missing id is accepted.